lib/util.test.js

```diff
@@ -58,16 +61,19 @@ describe("Testing cleanAnonymous function", () => {
     expect(pg.deletePgAccount).not.toHaveBeenCalled();
     expect(es.deleteAccount).not.toHaveBeenCalled();
     expect(arango.deleteAccount).not.toHaveBeenCalled();
+    expect(ignite.deleteAccount).not.toHaveBeenCalled();
   });
   it("should call database functions if expired accounts are found", async () => {
     Accounts.findAll.mockReturnValue([{ destroy: () => {} }]);
     pg.userHasPgAccount = () => true;
     es.checkAccount = () => true;
     arango.checkIfDatabaseExists = () => true;
+    ignite.checkAccount = ()=>true;
```

Comment:
Can you change the format to include spaces?

`ignite.checkAccount = () => true`